mlir/test/Target/LLVMIR/llvmir-le-specific.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: mlir-translate -mlir-to-llvmir -split-input-file %s | FileCheck %s
+
+// Decoding the attribute does not work on big-endian platforms currently
+// XFAIL: target=s390x-{{.*}}
```

Comment:
LLVM tests use "host-byteorder-little-endian"; maybe we can do the same thing here?  (You might need to modify the MLIR lit.cfg.py.)

Not sure how much this matters, given the decline of big-endian PPC, but it would make the intent clearer.